tests/rptest/tests/acls_test.py

```diff
@@ -256,25 +313,11 @@ def should_pass_w_authn_user(use_tls: bool, use_sasl: bool,
         pass_w_authn_user = should_pass_w_authn_user(use_tls, use_sasl,
                                                      enable_authz, client_auth)
 
-        # run a few times for good health
-        for _ in range(2):
-            try:
-                self.get_client("base").acl_list()
-                assert pass_w_base_user, "list acls should have failed for base user"
-            except ClusterAuthorizationError:
-                assert not pass_w_base_user
-
-            try:
-                self.get_client("cluster_describe").acl_list()
-                assert pass_w_authn_user, "list acls should have failed for cluster user"
-            except ClusterAuthorizationError:
-                assert not pass_w_authn_user
-
-            try:
-                self.get_super_client().acl_list()
-                assert pass_w_authn_user, "list acls should have failed for super user"
-            except ClusterAuthorizationError:
-                assert not pass_w_authn_user
+        self.check_permissions(pass_w_base_user=pass_w_base_user,
+                               pass_w_cluster_user=pass_w_authn_user,
+                               pass_w_super_user=pass_w_authn_user,
+                               timeout_sec=20,
```

Comment:
These timeouts vary a lot. How are they chosen? Does making them longer slow down the tests?

Reply:
> These timeouts vary a lot. How are they chosen? Does making them longer slow down the tests?

No that was a typo that I thought I removed. I had different timeouts for quicker debugging purposes.
Fixed.